build/configs/rtl8721csm/security_hal_test_tz/defconfig

```diff
@@ -388,6 +388,11 @@ CONFIG_USERMAIN_STACKSIZE=2048
 CONFIG_PTHREAD_STACK_MIN=256
 CONFIG_PTHREAD_STACK_DEFAULT=2048
 
+#
+#SRAM Heap size information
+#
+CONFIG_SRAM_HEAP_SIZE=28
```

Comment:
The available heap for this config before patch is "29984". So the "CONFIG_SRAM_HEAP_SIZE" value can only be maximum of (available heap/1024) or else we will get a overflow compile error. Then may need to adjust accordingly.